addons/common/XEH_postInit.sqf

```diff
@@ -153,6 +153,77 @@
     _unit enableIRLasers _mode;
 }] call CBA_fnc_addEventHandler;
 
+[QGVAR(forceFire), {
+    params ["_curatorClientID", "_shooters"];
+
+    _shooters = _shooters select {local _x};
+    if (_shooters isEqualTo []) then {
+        GVAR(forceFireCurators) = GVAR(forceFireCurators) - [_curatorClientID];
+    } else {
+        // track which curators are forcing fire
+        GVAR(forceFireCurators) pushBackUnique _curatorClientID;
+        [{
+            params ["_args", "_pfhID"];
+            _args params ["_curatorClientID", "_shooters", "_startTime"];
+
+            if (!(_curatorClientID in GVAR(forceFireCurators)) || {CBA_missionTime - _startTime > 10}) exitWith {
```

Comment:
Change to `_endTime` so then the check is just `CBA_missionTime >= _endTime`.

Reply:
32badb6